securedrop/tests/test_db.py

```diff
@@ -78,3 +78,10 @@ def test_journalist_string_representation(self):
     def test_source_string_representation(self):
         test_source, _ = db_helper.init_source()
         test_source.__repr__()
+
+    def test_throttle_login(self):
+        journalist, _ = db_helper.init_journalist()
+        for _ in range(Journalist._MAX_LOGIN_ATTEMPTS_PER_PERIOD):
+            Journalist.throttle_login(journalist)
+        with self.assertRaises(LoginThrottledException):
```

Comment:
Since this exception is unique to this error condition, I did not think of using assertRaisesRegexp. Could you please explain why it would be better ?